qa/supply-chain/audits.toml

```diff
@@ -675,6 +728,12 @@ who = "Jack Grigg <jack@electriccoin.co>"
 criteria = "safe-to-deploy"
 delta = "1.0.111 -> 1.0.113"
 
+[[audits.der]]
+who = "Daira-Emma Hopwood <daira@jacaranda.org>"
+criteria = "safe-to-deploy"
+delta = "0.7.8 -> 0.7.9"
+notes = "The change to ignore RUSTSEC-2023-0071 is correct for this crate. In any case we do not use RSA."
```

Comment:
These audits are not just Zcash-specific; they will be broadcast to other `cargo-vet` users who subscribe to our audits.
```suggestion
notes = "The change to ignore RUSTSEC-2023-0071 is correct for this crate."
```